src/components/otpInput/OTPInput.tsx

```diff
@@ -91,6 +93,12 @@ export const OTPInput = React.forwardRef<View, Props>(
       handleValidate(value);
     };
 
+    React.useEffect(() => {
+      if (autoFocus) {
+        setHasFocus(true);
+      }
+    }, [autoFocus]);
```

Comment:
Instead of working with effects, why not changing the state definition at line 55 using autoFocus value to define the hasFocus state ?

Reply:
Good catch! I'll try it out.

Reply:
Done! [c73a497](https://github.com/pagopa/io-app-design-system/pull/252/commits/c73a497bb1147b910949a2d334336d2281fd66fc)